doc/concepts/data-concepts/repo.md

```diff
@@ -0,0 +1,33 @@
+# Repository
+
+A Pachyderm repository is a location where you store your data inside
+Pachyderm. A Pachyderm repository is a top-level data object that contains
+files and folders. Similar to Git, a Pachyderm repository tracks all
+changes to the data and creates a history of data modifications that you
+can access and review. You can store any type of file in a Pachyderm repo,
+including binary and plain text files.
+
+Unlike a Git repository that stores history in a `.git` file in your copy
+of a Git repo, Pachyderm stores the history of your commits in a centralized
+location in the etcd key-value store. Because of that, you do not run into
+merge conflicts as you often do with Git commits when you try to merge
+your `.git` history with the master copy of the repo. With large datatsets
+resolving a merge conflict might not be possible.
+
+A Pachyderm repository is the first entity that you configure to create
+a standard pipeline. You can create a repository by running the
```

Comment:
```suggestion
A Pachyderm repository is the first entity that you configure when you want to add data to Pachyderm
```

Technically, creating the repo has nothing to do with the pipeline. They could be using pachyderm for jsut data versioning and never use pipelines.